plain-pipelines.d/paas-product-page.yml

```diff
@@ -0,0 +1,114 @@
+---
+resource_types:
+  - name: s3-iam
+    type: docker-image
+    source:
+      repository: governmentpaas/s3-resource
+      tag: 594eaa9f4d93b2ed32a7e5e1cdea5b380e2f6682
+
+resources:
+  - name: paas-product-page
+    type: git
+    source:
+      uri: https://github.com/alphagov/paas-product-page.git
+      branch: master
+
+  - name: deskpro-secrets
+    type: s3-iam
+    source:
+      bucket: ((state_bucket))
+      region_name: ((aws_region))
+      versioned_file: deskpro-secrets.yml
+
+  - name: zendesk-secrets
+    type: s3-iam
+    source:
+      bucket: ((state_bucket))
+      region_name: ((aws_region))
+      versioned_file: zendesk-secrets.yml
+
+jobs:
+  - name: deploy
+    serial: true
+    plan:
+      - get: paas-product-page
+        trigger: true
+      - get: deskpro-secrets
+      - get: zendesk-secrets
+      - task: push
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              # FIXME
```

Comment:
what is this FIXME here?

Reply:
This is as commented in the PR description: 

> - Merge and generate the container for alphagov/paas-docker-cloudfoundry-tools#124
> - Update the corresponding commit in this branch